add missing `await`s for `getPackageVersion` calls

dario-piotrowicz opened the pull request:
### Before
![Screenshot 2023-06-16 at 09 46 07](https://github.com/cloudflare/next-on-pages/assets/61631103/6c1ec921-7150-4baa-a799-07927a76f378)

### After
![Screenshot 2023-06-16 at 09 46 25](https://github.com/cloudflare/next-on-pages/assets/61631103/cc89907a-9b20-47aa-bd63-62fade7c3f9c)

___

Also
resolves #305

petebacondarwin reviewed (approved):
We should have an eslint warning on async expressions that are not awaited...

dario-piotrowicz commented:
@petebacondarwin I've added some eslint rules to help us with promises

The rule for forcing `await` is [`no-floating-promises`](https://typescript-eslint.io/rules/no-floating-promises/): 
![Screenshot 2023-06-21 at 15 48 46](https://github.com/cloudflare/next-on-pages/assets/61631103/34308f54-14fb-4a1f-a466-3fe79550055a)

But unfortunately I think it can't handle promises generated in other files 😕, so it's not really helping us too much there 😓 
![Screenshot 2023-06-21 at 15 45 38](https://github.com/cloudflare/next-on-pages/assets/61631103/96b06415-22b3-41e5-8608-fcf66f996415)
(👆 the rule is set and I've removed the `await` but there's no warning 😓)

At the very least these additions should anyways be helpful even though not a complete silver bullet 🙂 

Please have another quick look to see if you're ok with the last changes 🙏
